CHANGES.md

```diff
@@ -1,6 +1,473 @@
+* changed the alignment of a non-stridable range from 0 to its low bound
+  (i.e., `(3..5).alignment` is now 3 where it used to be 0)
+
+New Features
+------------
+* added support for uninterpreted triple-quoted strings
+  (see 'Literals' section in 'Lexical Structure' chapter of the specification)
+* added support for `continue` statements within forall loops
+* extended `delete` to accept arrays and list of expressions to delete
+  (see 'Deleting Class Objects' in the 'Classes' chapter of the specification)
+* added an `isEmpty()` method on domains
+* type and param fields can now be accessed from a type variable
+  (see 'Field Accesses' in the 'Classes' chapter of the specification)
+* supported forwarding method calls on arrays, domains, and distributions
+  (see https://chapel-lang.org/docs/1.17/technotes/dsi.html#overview)
+* added support for associative domains whose index types contain ranges
+* added support for subtype queries on distributions
+  (see 'Methods on All Domain Types' in the 'Domains' chapter of the spec)
+* added support for using GASNet's `smp` conduit
+
+Feature Improvements
+--------------------
+* dramatically improved support for user-defined initializers
+  (see https://chapel-lang.org/docs/1.17/technotes/initializers.html)
+* made `real(32)` variables significantly more usable
+  (see 'Implicit Numeric, Bool, and Enumeration Conversions' in the spec)
+* made string->value casts throw rather than halt() when encountering errors
+* `Owned(c)` and `Shared(c)` now coerce to type `c`
+  (see https://chapel-lang.org/docs/1.17/modules/packages/OwnedObject.html)
+* `Owned(d)`/`Shared(d)` now coerce to `Owned(c)`/`Shared(c)` for `class d: c`
+  (see https://chapel-lang.org/docs/1.17/modules/packages/OwnedObject.html)
+* writing out an `Owned(c)` now simply prints the `c` object
+* improved support for casting arrays to strings
+* made clear() for an array of records call the records' deinitializers
+* added support for defining multiple config types in a single statement
+  (see 'Type Aliases' in the 'Types' chapter of the language specification)
+* made error-handling propagate through forwarded methods
+* added support for invoking recursive parallel iterators via forall loops
+* enabled wide pointers to be cast to `c_void_ptr`
+* improved support for enums with non-trivial initialization expressions
+* improved default argument handling
+
+Removed Features
+----------------
+* removed the `msg` field from the `Error` class
+* removed deprecated `Barrer` module in favor of `Barriers`
+* removed deprecated `LayoutCSR` module in favor of `LayoutCS`
+* removed `ReplicatedDist` in favor of `Replicated`
+
+Standard Modules/Library
+------------------------
+* added Bessel fucntions to `Math` module
+  (see https://chapel-lang.org/docs/1.17/modules/standard/Math.html#Math.j0)
+* added support for joinPath(), isAbsPath(), commonPath to the `Path` module
```

Comment:
commonPath does not have parentheses when the other two do, I think they should follow the same strategy